lms/templates/discussion/_switch_experience_fragment.html

```diff
@@ -7,9 +7,9 @@
 %>
 
 % if show_banner:
-<div class="upgrade-banner d-flex bg-primary text-white align-items-center px-4 py-3">
+<div class="upgrade-banner d-flex bg-primary text-white align-items-center px-4">
     % if show_mfe:
-    <div class="d-flex w-100">
+    <div class="d-flex w-100 text-">
```

Comment:
nit: `text-`
Do you want to add any text class?

Reply:
that was left by mistake